### DISCUSSION TITLE
Crowdfunding for Ravencoin kawpow support

### DISCUSSION THREAD

**Discussion by Seal-Clubber** [coinfoundry/miningcore#1334] [General]
The Ravencoin community is looking for an MIT licensed kawpow pool that supports RVN. For this the community is prepared to have a bounty/crowdfunding to accelerate this development. I assume (@oliverw , @Konstantin35 ) would know a fair estimation about the funding amount to achieve this goal.

**Comment by Seal-Clubber** [comment]
The Ravencoin foundation has made a post about this subject under "Open Source Mining Bounties" - Pool. Feel free to contact them through the email provided.

https://ravencoin.foundation/proposal_desc/

**Reply by oliverw** [reply]
Interesting. Will take a look.
